hazelcast/src/main/resources/hazelcast-full-example.yaml

```diff
@@ -1559,15 +1557,17 @@ hazelcast:
         max-size-policy: ENTRY_COUNT
         eviction-policy: LFU
       wan-replication-ref:
-         name: my-wan-cluster-batch
-         merge-policy: com.hazelcast.cache.merge.PassThroughCacheMergePolicy
-         republishing-enabled: true
-         filters:
-           - com.example.SampleFilter
+        name: my-wan-cluster-batch
+        merge-policy: com.hazelcast.cache.merge.PassThroughCacheMergePolicy
+        republishing-enabled: true
+        filters:
+          - com.example.SampleFilter
       quorum-ref: quorumRuleWithThreeNodes
       partition-lost-listeners:
         - com.your-package.YourPartitionLostListener
-      merge-policy: com.hazelcast.cache.merge.LatestAccessCacheMergePolicy
+      merge-policy:
+        batch-size: 100
+        class-name: com.hazelcast.cache.merge.LatestAccessCacheMergePolicy
```

Comment:
The class `com.hazelcast.cache.merge.LatestAccessCacheMergePolicy` does not exist any more.

Reply:
fixed.